tests/pytorch/test_nn.py

```diff
@@ -682,6 +682,29 @@ def test_gin_conv(g, idtype, aggregator_type):
     gin = gin.to(ctx)
     h = gin(g, feat)
 
+@parametrize_dtype
+@pytest.mark.parametrize('g', get_cases(['homo', 'block-bipartite'], exclude=['zero-degree']))
```

Comment:
Does the module work for zero-degree nodes?

Reply:
It works for zero-degree nodes. I've changed the unit test.